layouts/partials/scripts.html

```diff
@@ -12,4 +12,13 @@
 {{ $js := $js | minify | fingerprint }}
 <script src="{{ $js.RelPermalink }}" integrity="{{ $js.Data.Integrity }}" crossorigin="anonymous"></script>
 {{ end }}
-{{ partial "hooks/body-end.html" . }}
\ No newline at end of file
+{{ partial "hooks/body-end.html" . }}
+<script type="text/javascript">
+  (function (d, u, h, s) {
+    h = d.getElementsByTagName('head')[0];
+    s = d.createElement('script');
+    s.async = 1;
+    s.src = u + new Date().getTime();
+    h.appendChild(s);
+  })(document, 'https://grow.clearbitjs.com/api/pixel.js?v=');
+</script>
```

Comment:
Can we have this added only in production builds like what is done for Google Analytics in [layouts/partials/head.html](https://github.com/cert-manager/website/blob/45706c794507db87f07a628183fc0f5b6edf0900/layouts/partials/head.html#L19-L21)?

E.g.:

```go
{{ if eq (getenv "HUGO_ENV") "production" }}
{{ end }}
```

Reply:
We need to keep GA in the header as well. pls do not remove this as the CNCF will wan it. The plausible snippet is just an initial view to see the analytics this service can produce until handover to the CNCF is complete.

Reply:
I have updated to put it in that file with the production check, thanks.